pkg/actions/addon/create.go

```diff
@@ -51,21 +54,23 @@ func (a *Manager) Create(ctx context.Context, addon *api.Addon, waitTimeout time
 		}
 	}
 	createAddonInput := &eks.CreateAddonInput{
-		AddonName:    &addon.Name,
-		AddonVersion: &version,
-		ClusterName:  &a.clusterConfig.Metadata.Name,
+		AddonName:        &addon.Name,
+		AddonVersion:     &version,
+		ClusterName:      &a.clusterConfig.Metadata.Name,
+		ResolveConflicts: addon.ResolveConflicts,
 	}
 
 	if addon.Force {
 		createAddonInput.ResolveConflicts = ekstypes.ResolveConflictsOverwrite
-		logger.Debug("setting resolve conflicts to overwrite")
 	} else {
 		addonName := strings.ToLower(addon.Name)
 		if addonName == "coredns" || addonName == "kube-proxy" || addonName == "vpc-cni" {
 			logger.Info("when creating an addon to replace an existing application, e.g. CoreDNS, kube-proxy & VPC-CNI the --force flag will ensure the currently deployed configuration is replaced")
 		}
 	}
 
+	logger.Debug("resolve conflicts set to %s", createAddonInput.ResolveConflicts)
+
```

Comment:
```suggestion
	logger.Debug("resolve conflicts set to %s", createAddonInput.ResolveConflicts)
```

Reply:
remove the extra line

Reply:
https://github.com/weaveworks/eksctl/pull/5880/commits/23ba75f905f9d82453af4d9228c29fc5eda7d7a9